src/css.js

```diff
@@ -263,8 +264,12 @@ jQuery.extend( {
 			// If a hook was provided, use that value, otherwise just set the specified value
 			if ( !hooks || !( "set" in hooks ) ||
 				( value = hooks.set( elem, value, extra ) ) !== undefined ) {
+				if ( typeof value === "string" && value.includes( "!important" ) ) {
 
-				if ( isCustomProp ) {
+					// Support !important priority (gh-3713).
+					var splitValue = value.split( "!" );
+					style.setProperty( argName,  splitValue[ 0 ], splitValue[ 1 ] );
```

Comment:
This may (usually *will*) leave trailing space on the value string, is that okay? Otherwise it should be trimmed.

Reply:
It should not be trimmed and I’d like to see a test that verifies that. The CSS var value is everything between the colon after the property name and “!important”. E.g. for the declaration:
```css
--foo: bar !important;
```
Running `getPropertyValue("--foo") on the matches element style returns `" bar "`, with both leading & trailing spaces.

Reply:
@dmethvin getPropertyValue will work as usual. It won't be affected by `!important`